rerun_py/src/python_bridge.rs

```diff
@@ -930,7 +930,7 @@ fn version() -> String {
 /// Get a url to an instance of the web-viewer
 ///
 /// This may point to app.rerun.io or localhost depending on
-/// whether `host_assets` was called.
+/// whether `start_web_viewer_server` was called.
```

Comment:
prevent doclinks from rotting again

```suggestion
/// whether [`start_web_viewer_server`] was called.
```